src/modules/layouts/utils.ts

```
@@ -36,4 +36,457 @@ export function compareValues(
+  switch (groupBy) {
+    case "relative":
+      if (diffDays <= 1) return "Today";
+      if (diffDays <= 7) return "This week";
+      if (diffDays <= 30) return "This month";
+      if (diffDays <= 365) return "This year";
+      return "Older";
+    case "day":
+      return date.toLocaleDateString();
+    case "week":
+      const weekStart = new Date(date);
+      weekStart.setDate(date.getDate() - date.getDay());
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)